Unit1/README.md

```diff
@@ -101,7 +106,7 @@
 ### Mini-Projects
 
 * Included in OC course:
-   * Mini-project 1.5: [Choose frameworks to build your API](https://openclassrooms.com/en/courses/6121936-build-your-web-projects-with-rest-apis/6268271-choose-frameworks-to-build-your-api)
```

Comment:
Make sure to update all the links, so they point to each README via a relative links.